src/navigator/NavigationService.ts

```diff
@@ -250,10 +247,3 @@ export function navigateToError(errorMessage: string, error?: Error) {
   Logger.debug(`${TAG}@navigateToError`, `Navigating to error screen: ${errorMessage}`, error)
   navigate(Screens.ErrorScreen, { errorMessage })
 }
-
-export function navigateToFiatCurrencySelection(flow: FiatExchangeFlow) {
```

Comment:
Wonder if we should retain this wrapper method similar to `navigateHome`. We've had instances in the past where we forgot to update all entry points for a flow (new send flow).

Reply:
Thought about keeping it, but as there's only navigation to the bottom sheet which handles cash in cash out it seemed superfluous: `navigate(Screens.FiatExchangeCurrencyBottomSheet, { flow: FiatExchangeFlow.<CashIn|CashOut> })`.